src/random_test.cpp

```diff
@@ -726,22 +772,26 @@ void Table::CreateDefaultColumn() {
       /* loop untill we select some column */
       while (col_type == Column::COLUMN_MAX) {
 
-        /* columns are 4:3:2:1:1 INT:VARCHAR:CHAR:BLOB:BOOL */
-        auto prob = rand_int(10);
+        /* columns are 6:2:2:2:2:2:1 INT:FLOAT:DOUBLE:VARCHAR:CHAR:BLOB:BOOL */
```

Comment:
varchar probability becomes very less . make it 4.

Reply:
Fixed